src/controllers/User.js

```diff
@@ -0,0 +1,99 @@
+const db = require("../models");
```

Comment:
Since `babel` is configured consider using modular `import` instead of `require`

Reply:
yeah i was lost let me work on it